app/code/Magento/Catalog/Ui/DataProvider/Product/Form/Modifier/Categories.php

```diff
@@ -52,6 +54,7 @@ class Categories extends AbstractModifier
      * @var array
      * @deprecated 101.0.0
      * @since 101.0.0
+     * @see Not use anymore
```

Comment:
`@see` used for linking to the class|function etc. In this case I prefer approve failed static test as legacy without changes like that

@sidolov what's your opinion?

Reply:
Can i use @codingStandardsIgnoreLine to ignore it?

Reply:
@Den4ik @sidolov  please advise

Reply:
I fixed